appium/options/ios/safari/safari_options.py

```diff
@@ -0,0 +1,293 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from appium.options.base_options_descriptor import OptionsDescriptor
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+
+class AutomaticInspectionOption(SupportsCapabilities):
+    AUTOMATIC_INSPECTION = 'safari:automaticInspection'
+    automatic_inspection = OptionsDescriptor(AUTOMATIC_INSPECTION)
+    """
+    This capability instructs Safari to preload the Web Inspector and JavaScript
```

Comment:
Please make sure docstrings follow Google style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I have followed numpy style documentation. This is also very nice.
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy

Reply:
It is good, although for this particular project we have already agreed to use the Google one. I don't think it would make sense to have multiple mixed styles there

Reply:
I will change the docstrings according to Google's standard.